src/Pages/Error/Error.jsx

```
@@ -1,31 +1,22 @@
 import React from "react";
+import { motion } from "framer-motion";
 import { Link } from "react-router-dom";
-import MatrixParallax from "react-matrix-parallax";
-
 import ms from "./Error.module.scss";
 
 export default function Error() {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)